packages/reactivity-transform/src/reactivityTransform.ts

```diff
@@ -550,6 +550,75 @@ export function transformAST(
     }
   }
 
+  /**
+   * unwrap the code form the macro($、$$), fix #6312 and keep the ideally behavior with the RFC#369
+   */
+  function unwrapMacro(node: CallExpression) {
+    const argsLength = node.arguments.length
+    const bracketStart = node.callee.end! + offset
+
+    s.remove(node.callee.start! + offset, bracketStart)
+
+    if (argsLength === 1) {
+      const bracketEnd = node.arguments[argsLength - 1].end! + offset
+      // remove brackets of macros
+      const argsType = node.arguments[0].type
+      if (
+        argsType === 'CallExpression' ||
+        argsType === 'Identifier' ||
+        argsType === 'ObjectExpression'
+      ) {
+        // fix space place
+        s.remove(node.start! + offset, node.arguments[0].start! + offset)
+        s.remove(bracketEnd, node.end! + offset)
```

Comment:
I think we could keep the spaces and comments
[Demo](https://deploy-preview-6944--vue-sfc-playground.netlify.app/#eNqlkMFqwzAQRH9lETmkIZJKCYWmaaH/oYtrb1IXWVpWawdj/O9dp6GU9pjjzKzmDZrMG5EbejR7cyg1tyRQUHp6DSmiQJ37JPAC988h1TmVHNHFfFpf/LuQVqs1wPXKb+ABNl61Bgf/3aY9KgQ7ipWgKoBpAsbSR4F5Xg5/QrM1bUeZxXYVuc+Sk66alifhGpRg9nBxFk9nLzqYDxEqe+8bpJhHS4xDi2f7+LTbWatXthxrq4zxxLq0cQkltsfRVUReY8c6v+3QYensO+dzQVZ8MNtfJK/mgGwZU4OMfDv5T+E/+gLXD5rN/AV9hJJQ)